Automattic/jetpack
# Add: Protect WAF header

#### Description
Adds a header section to the Protect plugin WAF UI, as per designs.

#### Changes proposed in this Pull Request:

* Adds a `FirewallHeader` component and applies it to the `FirewallPage`
* Adds an upgrade prompt
* Adds a `Popover` for the upgrade prompt details

#### To-Do
- [ ] Use `useWafData` hook for determining the `on`, `off` or `loading` status


#### Other information:

#### Jetpack product discussion


- 1201069996155224-as-1203282049665706

#### Does this pull request change what data or activity we track or use?


- No

#### Testing instructions:





* Start Jurassic Tube instance using this Protect branch
* Load the Firewall page and verify the `FirewallHeader` component displays as expected in all applicable situations (in both desktop and mobile and viewports in between): 
   * Note: At the moment, this can be achieved by manually setting the following component variables:
      * `status ={ 'on' }`
      * `status ={ 'off' }, hasRequiredPlan ={ 'true' }`
      * `status ={ 'off' }, hasRequiredPlan ={ 'false' }`
         * For this display: 
            * Hover over the `help` icon and ensure the `Popover` displays as expected
            * Ensure the upgrade button provides for a smooth checkout flow
      * `status ={ 'loading' }`
